Test/Tests/RestClientTests.swift

```diff
@@ -1822,7 +1822,7 @@ class RestClientTests: XCTestCase {
         let url = try XCTUnwrap(request.url, "No request url found")
         let acceptHeaderValue = try XCTUnwrap(request.allHTTPHeaderFields?["Accept"], "Accept HTTP Header is missing")
         
-        XCTAssertEqual(request.httpMethod, "PATCH")
+        XCTAssertEqual(request.httpMethod!.uppercased(), "PATCH")
```

Comment:
I don't understand this second change. What's it trying to achieve? I also can't replicate the failure locally — it passes using `"PATCH"` as the second argument for me.

Reply:
In Xcode 15 `XCTAssertEqual(request.httpMethod, "patch")` fails with `Optional("PATCH")") is not equal to ("Optional("patch")`. So I've changed it in 028cde0c0f1518fada4a0c81e5bb1b867ec04407 to uppercased "PATCH".

But on CI in Xcode 14.2 it began to fail with opposite message `Optional("patch")") is not equal to ("Optional("PATCH")`.

Thus to satisfy both Xcodes I've forced to use uppercase in this assertion.

Reply:
This goes into a separate [PR](https://github.com/ably/ably-cocoa/pull/1808)